crates/simple-git/Cargo.toml

```diff
@@ -13,11 +13,21 @@ license = "Apache-2.0 OR MIT"
 [dependencies]
 compact_str = "0.7.0"
 derive_destructure2 = "0.1"
-gix = { version = "0.52.0", features = ["blocking-http-transport-reqwest"] }
 thiserror = "1.0.40"
 tokio = { version = "1.30.0", features = ["rt", "time"], default-features = false }
 tracing = "0.1.37"
 
+[dependencies.gix]
+version = "0.53.1"
+default-features = false
+features = [
+    "comfort",
+    "progress-tree",
```

Comment:
Can you check with `cargo tree -e features -i gix-trace` and see if `git-trace` has the `tracing` feature enabled. Probably not, as this is not exposed in `gix` and I would be hesitant for now to do that as `gix-features` is meant for enabling features of all kind. Otherwise, `gix` would have to duplicate all `gix-features`… and maybe that will happen one day for convenience as it already does so for selected performance features.

[Here is](https://github.com/Byron/gitoxide/blob/46d0890a9aee1c2ab126614a5054a5c66293a2cc/Cargo.toml#L109) how `gitoxide` (the CLI) sets up tracing support.

Your feedback is definitely valued as you might be the only one, besides `gitoxide` itself, who integrates with the `tracing` crate.

Reply:
Sorry for the confusion, @Byron what I actually mean is that there's no progress output from gix.

I use tracing to log progress output from gix that's why I said "progress tracing", but gix actually has tracing support.

Reply:
That's interesting! There is no feature toggle to set here, progress depends solely on the functioning of the `NestedProgress` implementations that are passed to the respective methods.

Since there is a custom implementation here, it should be able to observe all calls coming from `gix`. From what I can tell, progress still works in `gix` (CLI) for example.

I hope that helps.

Reply:
Then perhaps our new progress impl is wrong, I would have to undo all features enabled and check our progress impl here.

Reply:
Yes, the correct course of action. 

`progress-tree` definitely isn't required unless you create a `NestedProgress` instance from it. No other feature will affect progress either.

Reply:
Thank you, turns out that I forgot to allow logging from simple-git.

This bug has been there for a while, ever since `simple-git` was extracted.